downloads/pt_PT/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: cheat-sheet
+title: Folheto de ajuda para Git do GitHub (pt-PT)
+byline: Git é um sistema de controlo de versão distribuído open source que facilita ações com o GitHub no seu portátil ou desktop. Este folheto de ajuda resume instruções frequentemente usadas na linha de comando do Git para referência rápida.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Instalar o Git
+
+Git é um sistema distribuído open source de controlo de versões que permite interagir com o GitHub no seu portátil ou desktop. Este folheto de ajuda resume as instruções frequentemente usadas na linha de comando do Git para referência rápida.
```

Comment:
@bitoiu There's this line, and there's the byline (line 4) in the `md`. Looking at the generated PDFs though, it appears there's only one place for this content and it's right below the title of the document. Since these two lines are different, is there a preference for which one to keep?  They're slightly different in wording.